### ISSUE TITLE
[20220323] Boostcamp-AI-Casual-Talk - 1회차

### ISSUE THREAD

**Issue by ili0820** [ili0820/Boostcamp-AI-Casual-Talk#2]
News
--
  - [스캐터랩, 인공지능 챗봇 '이루다1.0' 역경 딛고 ‘이루다2.0’ 오픈 베타 테스트 진행](http://www.aitimes.kr/news/articleView.html?idxno=24508&fbclid=IwAR0mJ45c68ZWQkLeO3jJJuCrn-f84-0Ke_SjG0G6qwh7fLt3byhRd-H0dZQ)</br>
    - [개인정보 불법수집·유출 논란 일었던 ‘이루다’, 1억330만원 과징금·과태료 부과](https://news.naver.com/main/read.naver?mode=LSD&mid=sec&sid1=105&oid=138&aid=0002102434&fbclid=IwAR2s-NGj7TgOBxeBUIvobJAiwXkmZOX3BNRKejj9Fzb2AU6kaZqQsCXx-WQ)
    
    - [‘이루다’ 사건에 대한 개인정보위 결정의 의미와 시사점](https://m.lawtimes.co.kr/Content/Article?serial=170076)
    - [[체험기]이루다 2.0 오픈…혐오 발화 줄었으나 특색 필요](https://www.etnews.com/20220316000078)
    - [인공지능의 법적 지위에 관한 논의*- 전자인(간)과 관련하여 -](http://lawresearch.cau.ac.kr/data/file/lawbooks/2780965405_lAn64RyP_458d27c22b1b1f761548b02350bf78602a3e0139.pdf)
    - [인공지능 시스템의 책임능력 ―전자인 제도의 도입 필요성에 관한 논의를 중심으로―](https://www.dbpia.co.kr/Journal/articleDetail?nodeId=NODE11022246) 
    - [스캐터랩 AI 챗봇 프라이버시 정책](https://scatterlab.co.kr/privacy#privacy)
<p align="center">
  <img width="700" height="286" src="https://user-images.githubusercontent.com/65278309/158923479-886e28ce-08ff-4491-a4b8-8563953f09c2.png">
</p>

<p align="center">
  <img width="644" height="409" src="https://user-images.githubusercontent.com/65278309/159284109-6451f56f-9c2d-4963-95c0-7ced8e8199aa.png">
</p>

  - [네이버클라우드, ‘인공지능 경진대회 플랫폼’ 사업 진출...AI 해커톤 솔루션 비롯해 지원 TF팀까지 구성 완료](http://www.aitimes.kr/news/articleView.html?idxno=24561&fbclid=IwAR3n3n0dfO68hFtrgkZ61XP3ozsPZWa-ZKFz9mdBKELz9mRW7ZD8T1jdaMM)
    - ["티라노사우루스와 이야기해볼까"…네이버가 만드는 새로운 포털 세상](http://www.aitimes.com/news/articleView.html?idxno=143525) 

<p align="center">
  <img width="600" height="577" src="https://user-images.githubusercontent.com/65278309/159283625-698daf58-cc04-4216-8917-fca73633c034.png">
</p>

  - [Stanford AI Index Report 2022](https://hai.stanford.edu/research/ai-index-2022)
    - [The State of AI in 9 Charts](https://hai.stanford.edu/news/state-ai-9-charts)
    1. What are the top skills most requested by employers?
    2. Employment of new AI PhDs in North America?
![image](https://user-images.githubusercontent.com/65278309/159310855-5a63a5bd-378d-4d28-95d4-e141bcdf8468.png)
    3. New U.S Resident AI Phds by Race/Ethnicity
    4. Private investment in AI
    5. Number of AI Publications by Field
    6. Number of AI Patent Filings
    7. A robot for everyone?
    8. Number of Accepted FAccT Conference Submissions by Affiliation
    9. Number of AI Related Bills in the United States
    

  - [When it comes to AI, can we ditch the datasets?](https://news.mit.edu/2022/synthetic-datasets-ai-image-classification-0315)
    -  [AI 학습에 데이터 세트 없어도 되나?...MIT, 합성 데이터 생성 모델 개발](http://www.aitimes.com/news/articleView.html?idxno=143485)
    - [GENERATIVE MODELS AS A DATA SOURCE FOR MULTIVIEW REPRESENTATION LEARNING](https://arxiv.org/pdf/2106.05258.pdf)
   
![image](https://user-images.githubusercontent.com/65278309/159283089-017b3ee9-8b54-4234-b939-40d6a6995122.png)

Arxiv
--
 - [FORCE: A Framework of Rule-Based Conversational Recommender System](https://arxiv.org/pdf/2203.10001v1.pdf)
 
![image](https://user-images.githubusercontent.com/65278309/159283896-fe49390c-161d-4b5b-96fc-01b8c183aa4c.png)

 - [Demo Video](https://www.youtube.com/watch?v=UjkLL5pgXx8)

**Comment by hawe66** [comment]
## Aesthetic preference for art can be predicted from a mixture of low- and high-level visual features
  
![Figure1](https://user-images.githubusercontent.com/67923359/159108366-0c4e63bf-bd5d-478d-bf76-26b12e1a7781.PNG)  
  
  
* The authors defined **low, high-level visual features** _(e.g.) hue, contrast / dynamic, concrete)_ and  
  proposed **Linear feature summation (LFS) model** that predicts human valuation of visual art  
  * LFS model successfully captures human liking ratings  
  
![Figure5](https://user-images.githubusercontent.com/67923359/159109053-37b2bcb0-1eb6-4e94-9c1a-989dbe5df12c.PNG)  
  
* **A deep convolutional neural network (VGG16) model predicts human liking ratings for visual art**  
  * VGG16: pre-trained by Imagenet / froze and performed feature-extraction on average art liking scores  
  * Ridge linear regression to predict liking ratings from each hidden layer  
  * **While lower layers of VGG16 capture the ratings of low-level features, higher layers of VGG16 capture the ratings of high-level features!**  
  
  
[Slides](https://docs.google.com/presentation/d/1QlrCT7ZXXs3ep_tFaWIkqmeTZ4k_L2me9QdjZOlj0FA/edit#slide=id.g11db2cd5f24_0_93)  
  
### News  
* GTC 2022

**Comment by gigio1023** [comment]
# OpenAI, Davinci model

https://andrewmayneblog.wordpress.com/2022/03/17/building-games-and-apps-entirely-through-natural-language-using-openais-davinci-code-model/

https://openai.com/blog/gpt-3-edit-insert/

## Overview
이전에 OpenAI에서 Codex를 통해 prompt로 코딩하는 모델을 선보였다. 이번에 새로운 GPT-3모델과 Codex가 나오면서 위 2개 글과 같은 결과를 얻을 수 있게 됐다. 

### New GPT-3 Capabilities: Edit & Insert
Edit
'아래 코드의 time complexity'를 개선시켜줘', 'javascript로 변환해줘'와 같이 prompt에 변화를 가할 수 있게 됐다.

Insert
Codex의 경우 앞, 뒤 prompt 맥락에 따라 적절한 코드가 지속적으로 중간 지점에 insert된다.
GPT-3로 Section별 글쓰기를 할 경우, Section2의 내용에 따라 Section1에 새로운 내용이 insert 된다.


# P-tuning

https://arxiv.org/pdf/2103.10385.pdf

## Overview
GPT 등에서 few-shot learning을 할 때 사용할 prompt를 hand-craft로 만들지말고, embedding으로 만들자.

## Method
e.g., NLU task에서 사용할 prompt를 embedding해보자.
The capital of **Britain** is [MASK].
사람이 직접 GPT에 넣을 prompt를 만든다면 아래와 같이 만들 수 있을 것이다.
![image](https://user-images.githubusercontent.com/11407756/159543422-57a5d1e5-76c8-4dcf-85de-772397085377.png)


이러한 작업을 사람이 실험을 통해 시행하지말고, PLM을 fine-tuning하지도 않으면서 prompt만 따로 embedding해보자.
![image](https://user-images.githubusercontent.com/11407756/159543019-7d284984-8c9c-4673-948b-222a155f1727.png)
- Prompt token: 학습해서 찾게 될 prompt들
- h: prompt encoder에 prompt token을 투입해 얻은 output

이렇게 만들어진 prompt를 PLM에 투입해 loss를 구하고 back propagation을 통해 prompt encoder를 업데이트한다.

## Result
![image](https://user-images.githubusercontent.com/11407756/159544127-f1ef0ee9-dba9-42e6-b334-cfb5896d058a.png)

일반적인 prompt 생성 방법들보다 더 좋은 결과가 나왔다고 한다.

**Comment by ijkimmy** [comment]
- News
    - [생각만으로 정확하게 로봇팔 조종한다!...KAIST 정재승 교수팀, '뇌-기계 인터페이스 인공지능' 모델 개발](http://www.aitimes.kr/news/articleView.html?idxno=24335)

    - [설명가능한 인공지능으로 의식의 깊이 측정...고려대-위스콘신대-리에주대 공동연구팀, 실시간으로 의식 수준 설명하는 AI 개발](http://www.aitimes.kr/news/articleView.html?idxno=24393)


    - [AI suggested 40,000 new possible chemical weapons in just six hours](https://www.theverge.com/2022/3/17/22983197/ai-new-possible-chemical-weapons-generative-models-vx)

- Blog Posts
    - [Building games and apps entirely through natural language using OpenAI’s code-davinci model](https://andrewmayneblog.wordpress.com/2022/03/17/building-games-and-apps-entirely-through-natural-language-using-openais-davinci-code-model/)

    - [Deep Neural Nets: 33 years ago and 33 years from now](https://karpathy.github.io/2022/03/14/lecun1989/)

- Research
    - [GopherCite: Teaching language models to support answers with verified quotes](https://deepmind.com/research/publications/2022/GopherCite-Teaching-Language-Models-To-Support-Answers-With-Verified-Quotes)

    - [Using Deep Learning to Annotate the Protein Universe](https://ai.googleblog.com/2022/03/using-deep-learning-to-annotate-protein.html?utm_source=pocket_mylist)

    - [ProteInfer: deep networks for protein functional inference](https://google-research.github.io/proteinfer/)

**Comment by sunbi-s** [comment]
# Web 3.0
- ### what is Web 3.0?
![image](https://user-images.githubusercontent.com/78129508/159693685-7b13c5a5-3ef5-4887-a88c-bc3b7a7dac4f.png)
- [Web3.0 ppt](https://github.com/ili0820/Boostcamp-AI-Casual-Talk/files/8332737/Web3.pptx)
